### PR TITLE
styles changed + cards fixed

### PR DESCRIPTION
**Pull request by Devesh21700Kumar**
**Issue Number**

fixes #

**Describe the changes you've made**

Fixes the cards font and sizes

added to lintignore

**Describe if there is any unusual behavior (Any Warning) of your code(Write `NA` if there isn't)**

**Additional context (OPTIONAL)**

**Test plan (OPTIONAL)**

A good test plan should give instructions that someone else can easily follow.

**Checklist**

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] The title of my pull request is a short description of the requested changes.

Readable Card below:-

image

image

also hover effects added to them

### CODE REVIEW

**Comment by Devesh21700Kumar** [comment]
@Abhishek-kumar09 i hope this is perfect this time.. 
would squash and rebase from next time if any merge conflicts are to be sorted

**Comment by Devesh21700Kumar** [comment]
@Abhishek-kumar09 this is the new PR. pls check the PR and merge

File: src/components/courseCard/courseCard1.js [hunk 1]
```diff
@@ -53,11 +62,9 @@ const CourseCard = () => {
           }}
         />
         <div style={{ backgroundColor: "#000", color: "#fff", padding: "2px 8px" }}>
-          <Typography variant="h6">Chapter 1 : How to start with WD?</Typography>
+          <Typography variant="h5">Chapter 1 : How to start with WD?</Typography>
```
Comment (Abhishek-kumar09) [inline src/components/courseCard/courseCard1.js:65]: PLease revert all the changes that you have done with variant. We are primarily developing this application for mobiles. If it looks too smaller on web we can do it with changing themes directory when building web apps.

Comment (Abhishek-kumar09) [reply src/components/courseCard/courseCard1.js:65]: This doesn't looks good: 
image

File: src/components/wishlistCard/index.js [hunk 2]
```diff
@@ -72,14 +87,18 @@ const WishlistCard = () => {
           }}
         />
         <CardContent>
-          <Typography>
+          <Typography style={{ fontSize: "14px" }}>
```
Comment (Abhishek-kumar09) [inline src/components/wishlistCard/index.js:90]: Please revert this

File: src/components/courseCard/courseCard1.js (hunk 1 above)
Comment (Devesh21700Kumar) [reply src/components/courseCard/courseCard1.js:65]: okay. so i'll make changes for mobile breakpoints then

Comment (Devesh21700Kumar) [reply src/components/courseCard/courseCard1.js:65]: image

the card view for mobile breakponts

Comment (Devesh21700Kumar) [reply src/components/courseCard/courseCard1.js:65]: image

cards view for mobile breakpoints

@Abhishek-kumar09 pls say if the above are fine or not

**Pull request merged**